handlebars/handlebars.d.ts

```diff
@@ -1,12 +1,66 @@
-// Type definitions for Handlebars 1.0
+// Type definitions for Handlebars v3.0.3

```

Comment:
I believe we should be preserving version 1.0 definitions as handlebars-1.0.d.ts. Then, you can keep this handlebars.d.ts as it is here.